hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -123,8 +123,13 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
   assign policy_${policy['name'].lower()}.write_perm = reg2hw.policy_${policy['name'].lower()}${"_shadowed" if enable_shadow_reg else ""}.write_perm.q;
 
 % endfor
+  localparam racl_policy_t UnusedPolicy = '0;
```

Comment:
Can we conditionally render the local parameter only when it is used in the later assignment? That fixes the lining issue.